src/microns_to_nwb/tools/behavior/behavior.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from hdmf.backends.hdf5 import H5DataIO
+from phase3 import nda
+from pynwb import TimeSeries
+from pynwb.behavior import PupilTracking, SpatialSeries, EyeTracking
+
+
+def add_eye_tracking(scan_key, nwb):
+    pupil_min_r, pupil_maj_r, pupil_x, pupil_y, timestamps = (nda.RawManualPupil() & scan_key).fetch1(
+        "pupil_min_r", "pupil_maj_r", "pupil_x", "pupil_y", "pupil_times"
+    )
```

Comment:
One question here is this represents an opportunity to map the DataJoint field names to to more human-readable ones. That is, `pupil_min_r` could be expanded to `pupil_minor_radius` (or radii, as from the paper, IDC https://bossdb-open-data.s3.amazonaws.com/iarpa_microns/minnie/functional_data/functional_data_technical_documentation_v3.pdf)

Reply:
Yes, and I would also separate this into two functions (as it was in the other PR) for adding eye position and pupil tracking data. I just wasn't sure whether the scope of this PR is strictly introducing the modules while keeping the methods from the notebook intact.